src/asset/asset-model.ts

```diff
@@ -0,0 +1,19 @@
+export interface AssetModel {
+  creator?: string;
+  createdTime?: string;
+  headline?: string;
+  initialTransaction?: string;
+  thumbnailUrl?: string;
+  explorerUrl?: string;
+  assetSourceType?: string;
+  captureTime?: string;
+  backendOwnerName?: string;
+  usedBy?: string;
+  captureEyeCustom?: Array<{
+    field: string;
+    value: string;
```

Comment:
How about setting the field and value as optional? Although they’re required in our spec, we haven't verified these values during commit.